libunwind/src/UnwindRegistersRestore.S

```diff
@@ -16,6 +16,8 @@
 
 #if defined(_AIX)
   .toc
+#elif defined(__aarch64__) && defined(__ELF__) && defined(_LIBUNWIND_EXECUTE_ONLY_CODE)
```

Comment:
`defined(__ELF__)` seems unnecessary. Perhaps LIBUNWIND_EXECUTE_ONLY_CODE could have a check to reject non-ELF platforms